Add a note about extra Google Colab setup step (#7886)
google/jax: README.md

pip install --upgrade pip
pip install "jax[tpu]>=0.2.16" -f https://storage.googleapis.com/jax-releases/libtpu_releases.html
```

> JAX comes pre-installed in Google Colab notebooks, but the following code must be used to initialize the TPU:

Reviewer: ```suggestion
Colab TPU runtimes come with JAX pre-installed, but before importing JAX you must run the following code to initialize the TPU:
```
